src/main/java/crazypants/enderio/conduit/TileConduitBundle.java

```diff
@@ -164,11 +164,15 @@ public void readCustomNBT(NBTTagCompound nbtRoot) {
     facadeMeta = nbtRoot.getInteger("facadeMeta");
 
     if(worldObj != null && worldObj.isRemote) {
-      if (conduits.size() == 1 && conduits.get(0) instanceof IItemConduit) {
+	  boolean stableConduit = true;
+	  for(IConduit iConduit: conduits) {
+	    if (!(iConduit instanceof IItemConduit) && !(iConduit instanceof IMEConduit) && !(iConduit instanceof IOCConduit)) stableConduit = false;
```

Comment:
Ender fluid conduits and energy conduits also shouldn't require this, in general I think it might be better to add a method with a default implementation to `IConduit` rather than hardcoding a list here

Reply:
So only redstone conduit needed ? it is easy to fix in this way

Reply:
redstone and non-ender fluid conduits (basic and pressurized) change their appearance dynamically